float64 accuracy
---
bool success
string message
